kore/src/Kore/Logger/Output.hs

```diff
@@ -183,13 +294,22 @@ parseKoreLogOptions =
     <*> (mconcat <$> many parseEntries)
     <*> parseDebugAppliedRuleOptions
     <*> parseDebugAxiomEvaluationOptions
+    <*> parseDebugSolverOptions
   where
+    parseType :: Parser KoreLogType
     parseType =
         option
             (maybeReader parseTypeString)
             $ long "log"
             <> help "Name of the log file"
-    parseTypeString filename = pure $ LogFileText filename
+
+      where
+        parseTypeString filename =
+            case filename of
+                "none" -> pure LogNone
```

Comment:
We recently removed the command-line options for `none` and `stderr` because they don't make sense. I would not like to add them back.